lib/screens/lyrics_screen.dart

```diff
@@ -326,6 +326,7 @@ class _LyricsViewState extends ConsumerState<LyricsView> with WidgetsBindingObse
                           (nextLine == null || currentMicros < nextLine.startMicros);
 
                       return Column(
+                        key: ValueKey(line),
```

Comment:
we could use `line.startMicros` here? the lyric text itself can repeat, but the timestamps should be unique?